app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
```

Comment:
- [ ] Please make sure to fix the issue where a user has signed up for the first use of the app. See the error image down below!
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/183249900-3af50998-e124-4275-aaaa-adb5f3b674d0.png)

Reply:
Error fixed 👍

Reply:
- [ ] I am still facing the same issue while signing up. Please fix this.

![Screenshot from 2022-08-06 23-00-17](https://user-images.githubusercontent.com/26767383/183260024-a1897ab3-f207-49f7-8aae-3eac0e996fa3.png)